web/src/features/panels/zone/FeedbackCard.tsx

```diff
@@ -0,0 +1,270 @@
+}) {
+  const inputPlaceholder = getQuestionTranslation('placeholder');
+  const optional = getQuestionTranslation('optional');
+  const text = getQuestionTranslation('input-question');
+
+  return (
+    <div>
+      <div className="flex justify-start">
+        <div className="pr-1 text-sm font-semibold text-black dark:text-white">
+          {optional}
+        </div>
+        <div
+          data-test-id="input-title"
+          className="text-sm font-normal text-black dark:text-white"
+        >
+          {text}
+        </div>
+      </div>
+      <textarea
+        data-test-id="feedback-input"
+        value={inputText}
+        onChange={(event) => {
+          handleInputChange(event);
+          calculateTextareaHeight(event);
+        }}
+        rows={1}
+        placeholder={inputPlaceholder}
+        className="my-2 w-full resize-none rounded border border-neutral-200 bg-transparent text-base focus:border-brand-green focus:ring-0 dark:border-gray-700"
+      />
+    </div>
+  );
+}
+
+function SubmitButton({ handleSave }: { handleSave: () => void }) {
+  const buttonText = getQuestionTranslation('submit');
+
+  return (
+    <Pill
+      classes="dark:hover:bg-gray-700 disabled:dark:text-gray-700 disabled:text-neutral-200 dark:text-white text-black hover:bg-neutral-200 border border-black dark:border-white disabled:dark:border-gray-700 disabled:border-neutral-200 h-9 w-full"
+      text={buttonText}
+      isButton={true}
+      onClick={handleSave}
+    />
+  );
+}
+
+function FeedbackActions({
+  state,
+  setState,
+  estimationMethod,
+}: {
+  state: string;
+  setState: Dispatch<SetStateAction<string>>;
+  estimationMethod?: string;
+}) {
+  const [inputText, setInputText] = useState('');
+  const [feedbackScore, setfeedbackScore] = useState('');
+
+  const question = getQuestionTranslation('rate-question');
+
+  const handleInputChange = (event: { target: { value: SetStateAction<string> } }) => {
+    setInputText(event.target.value);
+  };
+
+  const handleSave = () => {
+    setState('3');
+    fetch(`https://hooks.zapier.com/hooks/catch/14671709/3l9daod/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        score: feedbackScore,
+        feedback: inputText,
+        reference: estimationMethod,
+      }),
+    });
+  };
+
+  if (state === '3') {
+    return null;
+  }
+
+  return (
+    <div className="flex flex-col">
+      <div data-test-id="feedback-question" className="text-sm">
+        {question}
+      </div>
+      <ActionPills setState={setState} setFeedbackScore={setfeedbackScore} />
+      {state === '2' && (
+        <div>
+          <div className="my-3 h-[1px] w-full bg-neutral-200 dark:bg-gray-700" />
+          <div>
+            <InputField inputText={inputText} handleInputChange={handleInputChange} />
+            <SubmitButton handleSave={handleSave} />
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function ActionPills({
+  setState,
```

Comment:
This could do with a more descriptive name, what state is being set?

Reply:
Do you think `setFeedbackState` is descriptive enough?